composer.json

```diff
@@ -15,7 +15,7 @@
     },
     "require": {
         "php": "^5.6 || ^7.0",
-        "friendsofphp/php-cs-fixer": "^2.4.0"
+        "friendsofphp/php-cs-fixer": "~2.5.0"
```

Comment:
Using the `~` operator to prevent updates beyond patches (which might fail the build because of newly added fixers).